mate-panel/panel-run-dialog.c

```diff
@@ -744,6 +763,23 @@ panel_run_dialog_find_command_idle (PanelRunDialog *dialog)
 					    -1);
 		}
 
+		gtk_tree_model_get (model, &iter, COLUMN_VISIBLE, &visible, -1);
+		if (visible && visible_program_idx < G_N_ELEMENTS (accelerator_key_mapping)) {
+			gtk_list_store_set (dialog->program_list_store,
+								&iter,
+								COLUMN_ACCELERATOR_MASK, (gint)accelerator_key_mapping[visible_program_idx].modifier,
+								COLUMN_ACCELERATOR_KEY_VALUE, accelerator_key_mapping[visible_program_idx].key_id,
+								-1);
+			g_hash_table_insert (accelerator_keys_to_tree_iter_map, GUINT_TO_POINTER(accelerator_key_mapping[visible_program_idx].key_id), GINT_TO_POINTER(visible_program_idx));
+			visible_program_idx++;
+		} else {
+			gtk_list_store_set (dialog->program_list_store,
+								&iter,
+								COLUMN_ACCELERATOR_MASK, (gint)GDK_MOD1_MASK,
```

Comment:
shouldn't the modifier be `0`? It doesn't actually matter though.

Reply:
note to clarify: this is just a suggestion, as this is hidden anyway, it really doesn't matter, it just made my eyebrows rise a tad when I saw it.

Reply:
Fixed (fixup 851f23981817a9a29670f6f31aed59cb115a9ee8)